Script/PRONTO.py

```diff
@@ -1119,6 +1120,24 @@ def update_clinical_tsoppi_file(InPreD_clinical_tsoppi_data_file,sample_id,if_ge
 		fw.close()
 
 
+def pdf_page_image_to_ppt(pdf_file,pages,output_ppt_file):
```

Comment:
I like that the function is relatively short and the purpose clear from the name. 👍